ui/src/theme/bannerStyles.js

```diff
@@ -0,0 +1,101 @@
+import styled from '@emotion/styled';
+import ReactMarkdown from 'react-markdown';
+
+import { BANNER_TYPES } from '../utils/constants';
+
+import base from 'theme';
+import { Row } from 'theme/system';
+
+const {
+  keyedPalette: { brandPrimary, valencia },
+} = base;
+
+export const BannerWrapper = styled('section')`
```

Comment:
ah, I see what you did here. 
based on my [previous comment](https://github.com/nci-hcmi-catalog/portal/pull/1078/files#r1887100878) this could better be an `article` (as in an item).
```suggestion
export const BannerWrapper = styled('article')`
```

Reply:
Yeah, I went back and forth on the structure/semantics of all of this. What I ended up with (after your suggestion catching the missed `section` for the Header wrapper 😛) was:
```html
<header>                                 <!-- Defines a `banner` landmark -->
  <section>Banner</section>              <!-- Repeat for as many banners as there are -->
  <section>HCMI Main Header</section>    <!-- Main visual header/nav -->
</header>
```

It was based off of a few sources, but the key takeaways I got from them were:
* Banners as `section`s seems to be accepted ([articles](https://medium.com/@web-accessibility-education/dos-and-donts-of-accessible-cookie-banners-2c1a3102df2f) + GitHub's [Primer system](https://primer.style/components/banner) uses `section`s)
* `header` element naturally gets the `banner` landmark: https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/banner.html
* Page content should be contained by landmarks, hence putting the banners inside the header: https://dequeuniversity.com/rules/axe/4.1/region


I do agree, though. It would be better to have the banners in some kind of list. I'd be open to doing a UL + LI setup as an enhancement.

Reply:
that's insightful. thanks for sharing those links, and we'll make sure to think more about this one when implementing it in Stage